addons/mail/controllers/thread.py

```diff
@@ -14,6 +14,14 @@ def mail_thread_data(self, thread_model, thread_id, request_list):
         thread = request.env[thread_model].with_context(active_test=False).search([("id", "=", thread_id)])
         return thread._get_mail_thread_data(request_list)
 
+    @http.route("/mail/thread/more_followers", methods=["POST"], type="json", auth="user")
+    def mail_thread_more_followers(self, thread_model, thread_id, after, limit=30):
```

Comment:
I don't think the "more" is useful. It's true that we are only using it to load more followers, but the function doesn't really need to be constrained to only that.
after should then be `None` by default.